tests/util/blockdev_utils.rs

```diff
@@ -0,0 +1,74 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+use libstratis::consts::SECTOR_SIZE;
+use libstratis::engine::strat_engine::engine::DevOwnership;
+use libstratis::engine::strat_engine::metadata::SigBlock;
+use std::fs::OpenOptions;
+use std::io::{Read, Seek, SeekFrom};
+use std::io::Write;
+use std::path::Path;
+use std::path::PathBuf;
+use util::test_result::TestError;
+use util::test_result::TestErrorEnum;
+
+use util::test_result::TestResult;
+
+macro_rules! try_clean_devs {
+	($val:expr) => {
+		assert!(match clean_blockdevs_headers($val) {
+		        Ok(_) => true,
+		        Err(e) => {
+		        	error!("Failed to clean header. {:?}", e);
+		        	false
+			    }
+			}
+		)
+    }
+}
+
+#[allow(dead_code)] // Needed to avoid warning on test build
+pub fn get_ownership(path: &PathBuf) -> TestResult<DevOwnership> {
+
+    let mut f = try!(OpenOptions::new()
+        .read(true)
+        .open(&path));
+
+    let mut buf = [0u8; 4096];
+    try!(f.seek(SeekFrom::Start(0)));
+    try!(f.read(&mut buf));
+
+    let ownership = match SigBlock::determine_ownership(&buf) {
+        Ok(ownership) => ownership,
+        Err(err) => {
+            let error_message = format!("{} for device {:?}", err, path);
+            return Err(TestError::Framework(TestErrorEnum::Error(error_message)));
+        }
+    };
+
+    Ok(ownership)
+}
+
+pub fn clean_blockdevs_headers(blockdev_paths: &Vec<&Path>) -> TestResult<()> {
```

Comment:
Seems like this has too many plurals in the name, ```clean_blockdev_headers``` might seem more natural. Also, it might be possible to substitute for this method a map and then a find, or something.